plugins/scaffolder/alpha-api-report.md

```diff
@@ -16,6 +20,14 @@ import { ScaffolderTaskOutput } from '@backstage/plugin-scaffolder-react';
 import { SubRouteRef } from '@backstage/core-plugin-api';
 import { TemplateEntityV1beta3 } from '@backstage/plugin-scaffolder-common';
 
+// @alpha
+export const CategoryPicker: () => JSX.Element | null;
+
+// @alpha (undocumented)
+export function ContextMenu(
```

Comment:
`@backstage/plugin-scaffolder-react`

Reply:
Probably look for a better name here too for where it's used etc.

Reply:
[`ContextMenu` imports three route refs](https://github.com/backstage/backstage/blob/310b781eac12b52a031442b77e12a57ed5bc39b0/plugins/scaffolder/src/next/TemplateListPage/ContextMenu.tsx#L32-L36) from within the same package.

Should I convert the three boolean args into route ref args?

Reply:
Yep, either that, or providing `onClick` handlers for each of them.

Reply:
And I think that might be outdated now as those routeRefs don't exist anymore, they're just the same `routeRefs` as of the release tomorrow :)